packages/hash/frontend/src/testUtils/testUtils.tsx

```diff
@@ -1,12 +1,12 @@
 import { MockedProvider, MockedResponse } from "@apollo/client/testing";
 import { ThemeProvider } from "@mui/material";
-// eslint-disable-next-line no-restricted-imports
+
 import { render, RenderOptions } from "@testing-library/react";
 
 import { theme } from "../theme";
 
 type CustomRenderOptions = RenderOptions & {
-  mocks?: MockedResponse[];
+  mocks?: Readonly<MockedResponse[]>;
```

Comment:
I added `as const` in `packages/hash/frontend/src/pages/__mocks__/signup.mock.ts` to avoid `SIGNUP_MOCKS[0]!`. We now need `Readonly` to be able to use `SIGNUP_MOCKS` as `mocks`.